### PR TITLE
fixed bug that alowed read only admins to edit projects

### PR DESCRIPTION
**Pull request by okanji**
The front end should disable the input fields for projects that the read only admin are not the PO or technical lead.

### CODE REVIEW

**Comment by okanji** [comment]
Hey @w8896699, I've fixed the merge conflicts and updated the snapshots so this should be good for a final review

File: web/src/views/ProfileEdit.tsx [hunk 1]
```diff
@@ -118,6 +129,30 @@ const ProfileEdit: React.FC = (props: any) => {
     },
   });
 
+  useEffect(() => {
+    async function projectBelongToUser() {
+      try {
+        // @ts-ignore
```
Comment (w8896699) [inline web/src/views/ProfileEdit.tsx:135]: what is this for?

**Review by w8896699 — APPROVED** [review]

File: web/src/views/ProfileEdit.tsx [hunk 2]
```diff
@@ -118,6 +129,30 @@ const ProfileEdit: React.FC = (props: any) => {
...
+        const userEmail = data?.email;
+
+        if (userRoles.includes('administrator')) {
+          setIsEditDisabled(false);
+        } else if (typeof userEmail !== 'undefined') {
+          const contactEmails = profileState.contactDetails.map((contact) => contact.email);
+          setIsEditDisabled(!contactEmails.includes(userEmail));
+        }
+      } catch (err) {
+        console.log(err);
```
[the remark this answers is not in the document]
Comment (okanji) [reply web/src/views/ProfileEdit.tsx:147]: I agree, have made the addition

**Review by okanji — COMMENTED** [review]
Thanks Billy, I have made the changes that you requested

**Pull request merged**